src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,30 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't add actor to database", e);
```

Comment:
The exception message in `DataProcessingException` should be specific to the operation being performed and the entity being affected. In this case, you should include 'actor' in the message to indicate that the issue occurred while adding an actor. For example: "Can't add actor " + actor + " to database". This provides more context when debugging.